### PR TITLE
substitue `aircraft` for `craft`, `drone`, or `quad`

### PR DESCRIPTION
**Pull request by nerdCopter**
- replace `[Dd]rone`, `[Qq]uad`, `[Cc]raft` with `[Aa]ircraft`
- messages only, not variables.

### CODE REVIEW

**Review by haslinghuis — COMMENTED** [review]
- unmanned aerial vehicle
- rotorcraft
- multirotor

**Comment by nerdCopter** [comment]
- i never see the term `rotorcraft`, IRL.
- wing might not be `multirotor`
- `UAV` maybe usable, but existing language uses `[Cc]raft` often, and thus was open to `[Aa]ircraft`.

I'm thinking remain `[Cc]raft` for now, kick the can down the road.

File: locales/en/messages.json [hunk 1]
```diff
@@ -4497,7 +4497,7 @@
...
+        "message": "The first of two D-term lowpass filters.<br /><br />In dynamic mode, filtering will be stronger at low throttle, with less filtering and less delay as throttle increases. This helps control motor grinding or unexpected flyways on arming, while reducing delay and propwash after takeoff. The transition from low to high cutoff will happen earlier, as throttle is increased, with higher D-term lowpass expo values.<br /><br />In static mode, the cutoff frequency is fixed.  This is not recommended for D filtering.<br /><br />Filter type defaults to PT1, though some users have used only a single dynamic Bicraft filter here, with no second PT1.<br /><br />Changes that result in less D filtering may cause serious motor overheating or flyaways on arming.",
```
Comment (haslinghuis) [inline locales/en/messages.json:4500]: - Craft is like vehicle a very generic term.
- As we do not have a `Bicraft` filter I think most messages should remain using `quad` as they won't be applicable for wings.
- Translators will have to update.

Comment (nerdCopter) [reply locales/en/messages.json:4500]: lol, i used `sed` rather too indiscriminately.

**Review by limonspb — APPROVED** [review]

**Comment by ctzsnooze** [comment]
I tend to prefer 'aircraft' as an abbreviation for 'model aircraft', rather than 'craft'.  Never liked 'craft' much.  No-one calls our quads 'craft', not in the real world.  We have 'wings' and we have 'quads'.  True, we have some hexacopter, tricopters and so on, but they are rare.  And we don't want to use 'copter' since 'quads' are not 'helis'.

I appreciate that we are also supporting wings much more, but personally unless the tip needs it, I'd say 'quad' still.  

Some tooltips may have little or no relevance to wings; those that do should speak explicitly about wings.  

I'd prefer that a general tooltip spoke about 'quads', and if there was no need to change the content, say the same about was the same for wings, then leave it at that.  

If however a wing has some special requirement or note, eg the new special CLI options for wings, then we speak about 'wings' in that tooltip.

Alternatively, if we get very clever, then maybe if the user chooses 'wing' for the mixer type, or the USE_WING DEFINE is included, then all instances of 'craft' or 'quad' become 'wing'.  That would be the 'right' way to go :-)

PS - when used in the form "to improve the craft's responsiveness', we need an apostrophe before the 's'.

**Comment by nerdCopter** [comment]
updated.
- fixed single possessive
- replaced `[Cc]raft` with `[Aa]ircraft` while removing plural `s`
- skipped language variable names.

if this PR is not important or undesired, happy to close it.

**Comment by haslinghuis** [comment]
Sorry for not approving. Agree with @ctzsnooze. IMO [air]craft is too generic like calling a race car a vehicle. Some messages apply to quads only. For wings we probably need other descriptions.

**Comment by nerdCopter** [comment]
no problem :)

**Comment by ctzsnooze** [comment]
Why closing?

**Comment by nerdCopter** [comment]
> Why closing?

- please click the 3-dots on my prior comment. closed based on feedback. 
- i personally do like `aircraft`, but if others have difference of opinion, i'll respect such.
- dynamic drone/quad/wing/aircraft, might be too cumbersome to code. (personally choose not to implement)

**Comment by haslinghuis** [comment]
IMO nothing wrong using `quad` as we could use `UAV` as general term. Think `aircraft` is used for real planes. But perhaps we do a poll on discord to get more opinions :)

**Comment by ctzsnooze** [comment]
Our local regulators refer to all the things we fly, whether wings, quads, multi-rotors, helis - as 'model aircraft'.

I agree with nerdCopter that the term `aircraft` is a very good generic term.  It truly covers all of the 'craft' that we fly, and makes it clear that we are dealing with `air-craft`, not boats or cars, which are also 'craft'.

`UAV` is a good alternative generic term, as an acronym for `Unmanned Aerial Vehicle`.  It has the advantage of brevity, and means 'unmanned', whereas `aircraft` does not.  It has the disadvantage that it has military implications, in the sense that a 'vehicle' is `a machine that is self-propelled and is designed to transport people or cargo`, whereas most Betaflight users do not fly to transport people or cargo.  So `UAV` to me strongly implies drones used either to transport merchandise or to drop bombs.  And that's not what we are about, primarily.

Hence, of the two, I prefer `aircraft` over `UAV`.

My take on `aeroplane` or `plane` is that those terms are specific to a form of `aircraft` that has wings.  The `plane` part refers to the planar aspect of the wing surface that generates lift.  And that we can call these `wings`.

My preference therefore would be:
- `aircraft` for generic statements that apply to all kinds of model aircraft, and for general use, in place of `quad` or `quadcopter` or `UAV`.
- `multirotor aircraft` or `multirotor` for  statements applicable only to quadcopters and the like
- `wing` for `winged aircraft`.

File: locales/en/messages.json [hunk 2]
```diff
@@ -6954,10 +6954,10 @@
         "message": "Personalization"
     },
     "craftName": {
-        "message": "Craft name"
+        "message": "Aircraft name"
     },
     "configurationCraftNameHelp": {
-        "message": "Can be show in logs, backup file names and the OSD.</br>Can be set via the <b>craft_name</b> CLI variable."
+        "message": "Can be show in logs, backup file names and the OSD.</br>Can be set via the <b>aircraft_name</b> CLI variable."
```
Comment (haslinghuis) [inline locales/en/messages.json:6960]:
```suggestion
        "message": "Can be show in logs, backup file names and the OSD.</br>Can be set via the <b>craft_name</b> CLI variable."
```

**Review by haslinghuis — APPROVED** [review]

**Comment by McGiverGim** [comment]
I'm late to that PR, sorry...
This automatic replacement in one language, has invalidated the 18% of translations... A lot of work for translators.
Please take this into account before any other bulk change in the messages file

**Pull request merged**